library/lib.mli

```diff
@@ -182,14 +182,16 @@ val variable_section_kind : Id.t -> Decl_kinds.binding_kind
 val section_instance : GlobRef.t -> Univ.Instance.t * Id.t array
 val is_in_section : GlobRef.t -> bool
 
-val add_section_variable : name:Id.t -> kind:Decl_kinds.binding_kind -> poly:bool -> unit
+val add_section_variable : name:Id.t -> kind:Decl_kinds.binding_kind -> unit
 val add_section_context : Univ.ContextSet.t -> unit
 val add_section_constant : poly:bool -> Constant.t -> Constr.named_context -> unit
 val add_section_kn : poly:bool -> MutInd.t -> Constr.named_context -> unit
 val replacement_context : unit -> Opaqueproof.work_list
 
 val is_polymorphic_univ : Univ.Level.t -> bool
 
+val check_same_poly : poly:bool -> unit -> unit
```

Comment:
Would you mind adding a docstring? While I can infer what the function does the name doesn't seem very descriptive to me.

Reply:
I'd put "section something" in the function name, what is unclear is the object being checked

Reply:
Changed